crates/matrix-sdk/src/room/futures.rs

```diff
@@ -201,6 +201,12 @@ impl<'a> IntoFuture for SendRawMessageLikeEvent<'a> {
             );
 
             let response = room.client.send(request, None).await?;
+
+            info!(
+                event_id = ?response.event_id,
```

Comment:
Please put the event ID into the span, just like we do with the transaction ID.

This is currently functionally the same, but depending on if we add more log lines, then the event ID will be part of the new log lines as well.

Reply:
Oh, and `info()`might be a bit too verbose.

Reply:
Thx @poljar 

I updated, the `event_id` is now added to the current span.
I also changed to debug.
So now if I want the trace to appear in the EIX, I have to enable the tracing level to `.debug` for `matrix_sdk::room::futures`  in `RustTracing.swift`. Is there a way to make that default for all apps? Or do I have also to modify EXA?

Reply:
Gah, `matrix_sdk` is logged at the `info` level. Indeed you would have to bump the things in the respective clients.

Let's just revert to`info` then, sorry about not checking this before.